examples/chip-tool/commands/pairing/OpenJointCommissioningWindowCommand.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Location of this file should be probably examples/jf-control-app/commands/pairing/

Reply:
Addressed in latest commit.